privacyidea/lib/eventhandler/base.py

```diff
@@ -140,6 +141,11 @@ def conditions(cls):
                           "not exist in the userstore anymore."),
                 "value": ("True", "False")
             },
+            CONDITION.TOKEN_IS_EXPIRED: {
+                "type": "str",
+                "desc": _("The token has an expired validity period"),
```

Comment:
That makes sense. I think it can be changed to "token_validity_period" instead.
That reminds me of unexpected use cases, and I think it should now be more detailed than just true and false.

I would like to update the "check_validity_period" to return -1 (before valid period), 0 (valid), and 1 (expired), and update the condition to reflect accordingly. This also requires to update other test calls (not too complicated).

What do you think?